avocado_vt/test.py

```diff
@@ -84,17 +84,16 @@ class VirtTest(test.Test):
 
     env_version = utils_env.get_env_version()
 
-    def __init__(self, methodName='runTest', name=None, params=None,
-                 base_logdir=None, job=None, runner_queue=None,
-                 vt_params=None):
+    def __init__(self, **kwargs):
         """
-        :note: methodName, name, base_logdir, job and runner_queue params
+        :note: methodName, name, base_logdir, job/config and runner_queue params
```

Comment:
This PR is about to make avocado-vt compatible with new version of avocado where Test class uses config, but avocado-vt still have to be compatible with  avocado LTS version where Test class uses job object. I understand that this might be confusing, so I will write better docstring with explanation.

Reply:
Oh I see. Yes, this is a good idea.